static/css/timetable/partials/exploration_modal.scss

```diff
@@ -374,13 +375,16 @@
   h4 {
     font-size: 14px;
     margin: 0;
+    display: inline;
+    overflow: hidden;
   }
 
   h5 {
     color: $g777;
     font-size: 12px;
     font-weight: 400;
     margin: 5px 0;
+    display: inline;
```

Comment:
if you're applying display inline to all instances then remove from here and put it above in the parent class

Reply:
hi didnt put in above parent because not all instances are inline